src/Metadata/Property/Factory/AnnotationPropertyCollectionMetadataFactory.php

```diff
@@ -22,7 +22,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class AnnotationPropertyNameCollectionFactory implements PropertyNameCollectionFactoryInterface
+final class AnnotationPropertyCollectionMetadataFactory implements PropertyNameCollectionFactoryInterface
```

Comment:
This should be the other way round, but shall we remove this class entirely since it's unused and doesn't work anyway...